### ISSUE TITLE
Allow providing custom Info.plist entities

### ISSUE THREAD

**Issue by maxkatz6** [egramtel/dotnet-bundle#9]
With this really helpful library application does not handles Info.plist file creation directly, thus it make create some difficulties when we actually need to add additional properties that weren't part of bundle information.
To avoid unnecessary xml rewrites I would want to set additional msbuild properties (items group with type info?) or use additional external configuration or Info.plist template.

**Comment by x2bool** [comment]
Yes, I think we could go with something like this: https://github.com/egramtel/dotnet-bundle/pull/8/files

```xml
<ItemGroup>
    <CustomProperty Include="name">
        <CustomPropertyName>name</CustomPropertyName>
        <CustomPropertyValue>value</CustomPropertyValue>
    </CustomProperty>
    <CustomProperty Include="name2">
        <CustomPropertyName>name2</CustomPropertyName>
        <CustomPropertyValue>value2</CustomPropertyValue>
    </CustomProperty>
</ItemGroup>
```

**Comment by az-oolloow** [comment]
Any updates on this? need this to add `NSDocumentsFolderUsageDescription`

**Comment by x2bool** [comment]
Hi! I don't know when I will be able to work on this. But I'll definitely accept Pull Request to fix this issue.
